openstack/keystone/templates/service-api.yaml

```diff
@@ -16,6 +16,7 @@ metadata:
     prometheus.io/port: "{{.Values.api.metrics.port}}"
     prometheus.io/targets: "{{ required ".Values.alerts.prometheus missing" .Values.alerts.prometheus }}"
 {{- end }}
+    {{- include "utils.linkerd.pod_and_service_annotation" . | indent 4 }}
```

Comment:
This template needs restructuring, because there will be no `annotations` key if `.Values.api.metrics.enabled` is false. Should be fine to move the `if` in line 13 down one line.